controllers/strategy/hypershift_strategy.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Red Hat
+
+package strategy
+
+import (
+	identitatemv1alpha1 "github.com/identitatem/idp-client-api/api/identitatem/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	clusterv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
+)
+
+func (r *StrategyReconciler) hypershiftPlacementStrategy(strategy *identitatemv1alpha1.Strategy,
+	authRealm *identitatemv1alpha1.AuthRealm,
+	placement *clusterv1alpha1.Placement,
+	placementStrategy *clusterv1alpha1.Placement) {
+	// placementStrategy.Spec.Predicates = placement.Spec.Predicates
+	// Append any additional predicates the AuthRealm already had on it's Placement
+	placementStrategy.Spec.Predicates = []clusterv1alpha1.ClusterPredicate{
+		{
+			RequiredClusterSelector: clusterv1alpha1.ClusterSelector{
+				ClaimSelector: clusterv1alpha1.ClusterClaimSelector{
+					MatchExpressions: []metav1.LabelSelectorRequirement{
+						hostedClusterRequirement(),
+					},
+				},
+			},
+		},
+		// {
+		// 	RequiredClusterSelector: clusterv1alpha1.ClusterSelector{
+		// 		LabelSelector: metav1.LabelSelector{
+		// 			MatchExpressions: []metav1.LabelSelectorRequirement{
+		// 				notGRCRequirement(),
+		// 			},
+		// 		},
+		// 	},
+		// },
+	}
+
+	for _, cp := range placement.Spec.Predicates {
```

Comment:
This all looks common with backplane strategy.. Should we refactor?

Reply:
Done